CHANGELOG.md

```diff
@@ -13,6 +12,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Update colors #355
 - Use Lerna to reorganize repo into a monorepo #353
 - Removes FormField `alwaysShowLabel` prop
+- Hide `print` prop
```

Comment:
Thanks for updating the Changelog. I think this needs a broader discussion with the design system WG in #452 about a long-term action.